cdap-ui/app/cdap/components/DataPrep/Directives/ExtractFields/UsingDelimiterModal/index.js

```diff
@@ -30,7 +30,7 @@ const DELIMITER_MAP = {
   'Tab': '\\t',
   'Pipe': '\\|',
   'Whitespace': '\\s+',
-  'Custom Separator': 'CUSTOM'
+  'Custom separator': 'CUSTOM'
```

Comment:
does this get displayed anywhere?

Reply:
Yeah this is actually used as a label as well